cli/Dockerfile

```diff
@@ -1,12 +1,16 @@
-FROM alpine:3.5
+FROM alpine:3.6
 MAINTAINER Kontena, Inc. <info@kontena.io>
 
-ADD https://get.docker.com/builds/Linux/x86_64/docker-1.10.3 /usr/local/bin/docker
+ADD https://download.docker.com/linux/static/stable/x86_64/docker-17.06.2-ce.tgz /tmp
```

Comment:
This way the tgz package will be "left behind" in the layers forever. The later `rm -rf /tmp/docker*` just masks it out. Why not add it in the `RUN` with `wget` or something?

Reply:
Fixed.